pkgs/applications/video/mpv/default.nix

```diff
@@ -24,6 +24,7 @@
 , waylandSupport ? stdenv.isLinux
   , wayland
   , wayland-protocols
+  , wayland-scanner
```

Comment:
How exactly it fixes cross compilation?

Reply:
```
mpv-aarch64-unknown-linux-gnu> Checking for Wayland                                                      : wayland-scanner not found
mpv-aarch64-unknown-linux-gnu> You manually enabled the feature 'wayland', but the autodetection check failed.
```

Reply:
I had fixed this in #187797, which is still in staging. Not sure what will happen now.

Reply:
It is still in staging-next and it will probably cause a merge conflict somewhere.